dlp/inspect_content.py

```diff
@@ -50,10 +51,29 @@ def inspect_string(project, content_string, info_types,
     # dictionaries (protos are also accepted).
     info_types = [{'name': info_type} for info_type in info_types]
 
+    # Prepare custom_info_types by parsing the dictionary word lists and
+    # regex patterns.
+    if custom_dictionaries is None:
+        custom_dictionaries = []
+    dictionaries = [{
+        'info_type': {'name': 'CUSTOM_DICTIONARY_{}'.format(i)},
+        'dictionary': {
+            'word_list': {'words': custom_dictionaries[i].split(',')}
+        }
+    } for i in range(len(custom_dictionaries))]
```

Comment:
Rather than using `for i in range(len())` can we iterate directly over the values and use the `enumerate()` builtin to get the numbers? (Same throughout)

Reply:
Done.